app/assets/stylesheets/playlists_find.css.sass

```diff
@@ -0,0 +1,30 @@
+.playlist_find
```

Comment:
name this file with an underscore `_` at the beginning so that the asset pipleline doesn't compile it into its own CSS file.